cyclonedx-lib/build.xml

```diff
@@ -136,6 +146,34 @@
                 </jar>
         </target>
 
+        <property name="testSBOMFile" location="build/testSBOM.json"/>
+                  <delete file="${testSBOMFile}"/>
```

Comment:
Move the "delete" to the approriate target, so we know when it gets called logically..